public/app/features/explore/Logs/LogsNavigation.tsx

```diff
@@ -250,5 +262,8 @@ const getStyles = (theme: GrafanaTheme2, oldestLogsFirst: boolean) => {
       align-items: center;
       margin-top: ${theme.spacing(1)};
```

Comment:
We can migrate these to the object format, but not here because this will be backported.